tests/unit_test/data_processor/catalog_sync_log_processor_test.py

```
@@ -0,0 +1,571 @@
+
+    def test_get_logs(self):
+        bot = 'test'
+        logs = list(CatalogSyncLogProcessor.get_logs(bot))
+        assert len(logs) == 4
+
+    def test_is_limit_exceeded_exception(self, monkeypatch):
+        bot = 'test'
+        try:
+            bot_settings = BotSettings.objects(bot=bot).get()
+            bot_settings.catalog_sync_limit_per_day = 0
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)